account_invoice_sale_origin_link/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Account Invoice Sale Origin Link",
+    "summary": "Add link to the Sale Order in invoice source document.",
+    "version": "13.0.1.0.0",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "category": "Accounting & Finance",
+    "depends": ["account_invoice_origin_link", "sale"],
+    "data": [],
+    "installable": True,
+    "maintainers": ["OriolVForgeFlow", "GuillemCForgeFlow"],
+}
```

Comment:
In my opinion, I would add an auto_install=True

Reply:
True! Thanks :)